.github/workflows/storybook_deploy_main.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy Main Storybook
+
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - '**[Ss]tories**'
```

Comment:
Isn't it just `webpack/stories` where we store everything?

Reply:
`<componentName>.stories.js` files are usually stored next to the component, so they can be in lots of places.

Reply:
Wouldn't it be sufficient to just list `webpack` then? I think the storybook can change if components change, even if the story itself doesn't change.